chore: Add changeset (#705)
SAP/ai-sdk-js: .changeset/dark-geese-laugh.md

---
'@sap-ai-sdk/document-grounding': patch
'@sap-ai-sdk/orchestration': patch
---

> [New Functionality] Add support for S3-based document grounding.

Reviewer: ```suggestion
[New Functionality] Support document grounding with SharePoint and S3 data repositories.
```
